packages/rrweb/src/record/observer.ts

```diff
@@ -477,9 +477,25 @@ function initInputObserver(
     }
   }
   const events = sampling.input === 'last' ? ['change'] : ['input', 'change'];
+
+  let timerId: ReturnType<typeof setTimeout>;
+
+  const wrappedEventHandler = (e: Event) => {
+    if (sampling.input !== 'debounce') {
```

Comment:
Don't we want to skip debounce for checkboxes and radios?

Reply:
It actually makes sense, thank you. Have updated the handler